distributed-databases/pinot/data.tf

```diff
@@ -0,0 +1,5 @@
+data "aws_eks_cluster_auth" "this" {
+  name = module.eks.cluster_name
+}
+
+data "aws_availability_zones" "available" {}
```

Comment:
remove this file and move the code to top of `main.tf` file

Reply:
Done